c_src/membrane_ffmpeg_swresample_plugin/converter_lib.c

```diff
@@ -1,16 +1,22 @@
+
 #include "converter_lib.h"
 
 char *lib_init(ConverterState *state, char from_s24le,
                enum AVSampleFormat src_sample_fmt, int src_rate,
                int64_t src_ch_layout, enum AVSampleFormat dst_sample_fmt,
-               int dst_rate, int64_t dst_ch_layout) {
+               int dst_rate, int64_t dst_ch_layout)
+{
   state->swr_ctx = NULL;
 
   struct SwrContext *swr_ctx = swr_alloc();
-  if (!swr_ctx) {
+  if (!swr_ctx)
+  {
     return "swr_alloc";
   }
 
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wimplicit-function-declaration"
```

Comment:
I think this could be solved by including `libavutil/opt.h` in `converter_lib.h` and adding `libavutil` as one of os_deps